File: backend/core/branding.py
"""Branding configuration for Ascend"""

class BrandingConfig:
    """Centralized branding constants"""
    
    APP_NAME = "Ascend"
    PRODUCT_NAME = "Ascend Diagrams"
    TAGLINE = "Diagram with Confidence"
    DESCRIPTION = "The modern UML diagramming tool for teams"
    COMPANY_NAME = "Ascend"
    
    COLORS = {
        "primary": "#6B46C1",
        "dark": "#1E1B4B",
        "secondary": "#8B5CF6",
        "light": "#F3E8FF",
        "success": "#10B981",
        "error": "#EF4444",
        "warning": "#F59E0B",
        "info": "#3B82F6"
    }
    
    SOCIAL = {
        "twitter": "https://twitter.com/ascendapp",
        "github": "https://github.com/ascendapp",
        "linkedin": "https://linkedin.com/company/ascendapp"
    }
    
    LINKS = {
        "privacy": "https://ascend.app/privacy",
        "terms": "https://ascend.app/terms",
        "docs": "https://docs.ascend.app",
        "blog": "https://blog.ascend.app",
        "contact": "https://ascend.app/contact"
    }
    
    SUPPORT_EMAIL = "support@ascend.app"
    WEBSITE = "https://ascend.app"

def get_branding():
    """Get branding configuration"""
    return {
        "appName": BrandingConfig.APP_NAME,
        "productName": BrandingConfig.PRODUCT_NAME,
        "tagline": BrandingConfig.TAGLINE,
        "description": BrandingConfig.DESCRIPTION,
        "companyName": BrandingConfig.COMPANY_NAME,
        "colors": BrandingConfig.COLORS,
        "social": BrandingConfig.SOCIAL,
        "links": BrandingConfig.LINKS,
        "supportEmail": BrandingConfig.SUPPORT_EMAIL,
        "website": BrandingConfig.WEBSITE
    }

File: src/constants/branding.js
export const BRANDING = {
  APP_NAME: 'Ascend',
  PRODUCT_NAME: 'Ascend Diagrams',
  TAGLINE: 'Diagram with Confidence',
  DESCRIPTION: 'The modern UML diagramming tool for teams',
  COMPANY_NAME: 'Ascend',
  SUPPORT_EMAIL: 'support@ascend.app',
  WEBSITE: 'https://ascend.app',
  
  COLORS: {
    PRIMARY: '#6B46C1',
    DARK: '#1E1B4B',
    SECONDARY: '#8B5CF6',
    LIGHT: '#F3E8FF',
    SUCCESS: '#10B981',
    ERROR: '#EF4444',
    WARNING: '#F59E0B',
    INFO: '#3B82F6'
  },
  
  SOCIAL: {
    TWITTER: 'https://twitter.com/ascendapp',
    GITHUB: 'https://github.com/ascendapp',
    LINKEDIN: 'https://linkedin.com/company/ascendapp'
  },
  
  LINKS: {
    PRIVACY: 'https://ascend.app/privacy',
    TERMS: 'https://ascend.app/terms',
    DOCS: 'https://docs.ascend.app',
    BLOG: 'https://blog.ascend.app',
    CONTACT: 'https://ascend.app/contact'
  }
};

export const FEATURE_FLAGS = {
  ASCEND_REBRAND: process.env.REACT_APP_FEATURE_ASCEND_REBRAND === 'true',
  I18N_ENABLED: process.env.REACT_APP_I18N_ENABLED === 'true',
  NEW_LOGO: process.env.REACT_APP_NEW_LOGO === 'true'
};
